lib/Workflow/Place/StatusInfo.php

```diff
@@ -111,7 +111,8 @@ private function getAllPlaces($subject, bool $visibleInHeaderOnly = false, strin
          * @var PlaceConfig $place
          */
         foreach ($places as $place) {
-            $uniquePlaces[$place->getPlace()] = $place;
+            $uniqueLabel = $place->getLabel().'__'.$place->getPlace();
```

Comment:
should that be really part of the PR? 
seems like something different

Reply:
Removed again.